docs/configuration.md

```diff
@@ -195,6 +195,19 @@ information about each option that appears later in this page.
         maxidle: 16
         maxactive: 64
         idletimeout: 300s
+    health:
+      storagedriver:
+        enabled: true
+        interval: 10s
+        threshold: 3
+      file:
+        - file: /path/to/checked/file
+          interval: 10s
+          threshold: 3
+      http:
+        - uri: http://server.to.check/must/return/200
```

Comment:
I could add `headers` as well. Do you think there are any others worth adding for the health check?